.github/workflows/staging-deployment.yml

```diff
@@ -0,0 +1,58 @@
+name: Staging deployment
```

Comment:
I think we can have different worflows for backend and frontend, as with this setup we have to add backend deploy here as well?

Reply:
I don't know if it is still not possible to use output from one workflow in another or maybe something has changed, but if not, maybe it would be easier to just have one workflow `frontend.yml` which will do `tests` as one job and then based on branch, deploy job to either staging or preview. Here is example with backend deploy https://github.com/Vizzuality/tota/blob/main/.github/workflows/backend.yml#L70-L74

Reply:
It seems to be possible: https://docs.github.com/en/actions/using-workflows/reusing-workflows

I'll try to make the tests reusable! Thanks for the feedback.

Reply:
Nice that they added that! :)